packages/lit-auth-client/src/lib/lit-auth-client.ts

```diff
@@ -97,7 +94,13 @@ export class LitAuthClient {
         );
       }
 
-      const supportedNetworks = ['cayenne', 'habanero', 'manzano', 'datil-dev'];
+      const supportedNetworks = [
```

Comment:
:nit: Can we replace this check and `throw()` with `isSupportedLitNetwork()` from `misc` rather than adding to this array?